backend/dataall/modules/mlstudio/db/mlstudio_models.py

```diff
@@ -10,16 +11,19 @@
 class SagemakerStudioDomain(Resource, Base):
```

Comment:
Do we need to use the Resource and Base classes? I don't know if storing tags and description makes sense at the moment. Also, I would store not only the environmentUri, but also the SamlGroupName that has created it (in this case it matches the env admin group)

Reply:
Added!